cmd/platform/main.go

```diff
@@ -141,9 +144,19 @@ func main() {
 	qrService := services.NewQrStoreService(cachex)
 
 	cfg.CredentialStatus.SingleIssuer = false
+
+	mediaTypeManager := services.NewMediaTypeManager(
+		map[iden3comm.ProtocolMessage][]string{
+			iden3commProtocol.CredentialFetchRequestMessageType:  {string(packers.MediaTypeZKPMessage)},
+			iden3commProtocol.RevocationStatusRequestMessageType: {"*"},
+		},
+		*cfg.MediaTypeManager.StrictMode,
+		cfg.MediaTypeManager.Disable,
```

Comment:
why is it disabled here?

Reply:
What do you mean? Disable is a boolean flag. By default is false